File: ExpressJS Fundamentals/05_Session_and_Authentication/models/Car.js
const mongoose = require('mongoose');

const carSchema = new mongoose.Schema({
    model: { type: mongoose.Schema.Types.String, required: true },
    image: { type: mongoose.Schema.Types.String, required: true },
    pricePerDay: { type: mongoose.Schema.Types.Number, required: true },
    isRented: { type: mongoose.Schema.Types.Boolean, default: false }
});

const Car = mongoose.model('Car', carSchema);
module.exports = Car;

File: C# OOP Basics/01_Defining_Classes/Google/Car.cs
public class Car
{
    public string Model { get; set; }
    public double Speed { get; set; }

    public Car(string model, double speed)
    {
        Model = model;
        Speed = speed;
    }

    public override string ToString()
    {
        return $"{this.Model} {this.Speed}";
    }
}
